products/sce/autoscaling/settings.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Autoscaling Settings'
+---
+
+To fine-tune how autoscaling works in SaladCloud, you can adjust the following settings:
+
+### **Desired Queue Length**
+
+- **Description**: The target queue length that triggers the autoscaler to adjust the number of container instances.
+- **Range**: Must be between 1 and 100.
+- **Example**: If you set this to 20, the system will aim to maintain a queue length of 20 messages before scaling up or
+  down.
+
+### **Minimum Replicas**
+
+- **Description**: The minimum number of container instances that the system will scale down to.
+- **Range**: Must be between 0 and 100.
+- **Example**: If you set this to 3, the system will never scale below 3 running instances, even during low demand
+  periods. If set to 0, the system will scale down to zero instances when there are no jobs in the queue, which means
+  you do not pay for any resources when the queue is empty. However, this may result in longer startup times when new
+  jobs are added to the queue.
+
+### **Maximum Replicas**
+
+- **Description**: The maximum number of container instances that the system will scale up to.
+- **Range**: Must be between 1 and 200.
```

Comment:
The maximum value is either 200 _or_ the maximum number of instance quota the user is allotted. For the majority of our customers, they're not allowed to run 200 instance containers.

Reply:
This was supposed to update according to the number of replicas that the customer has available in quota, but I will need to double check it. Great callout.